docs/Register-Resource-Application.md

```diff
@@ -34,13 +34,15 @@ A resource application has an identifier URI (Application ID URI), which clients
 
 ### Define application roles
 
-The Azure API for FHIR and the OSS FHIR Server for Azure use [Azure Active Directory application roles](https://docs.microsoft.com/azure/architecture/multitenant-identity/app-roles) for role-based access control. To define which roles should be available for your FHIR Server API, open the resource application's [manifest](https://docs.microsoft.com/azure/active-directory/active-directory-application-manifest/):
+The Azure API for FHIR and the OSS FHIR Server for Azure use [Azure Active Directory application roles](https://docs.microsoft.com/azure/architecture/multitenant-identity/app-roles) for role-based access control. To define which roles should be available for your FHIR Server API, you can edit the resource application's [manifest](https://docs.microsoft.com/azure/active-directory/active-directory-application-manifest/) or add App roles from the portal. To add app roles from the portal
 
-1. Click **Manifest**:
+1. Click **App roles | Preview**
+1. Select **Create app role**
+1. Add the details from the [roles](https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Shared.Web/roles.json) file.  This includes globalAdmin, globalWriter, globalReader, and globalExporter. Select Both fo Allowed member types.
 
-    ![Application Roles](images/resource-application/portal-aad-register-new-app-registration-APP-ROLES.png)
+![Add App Roles](images/resource-application/approles.png)
 
-2. In the `appRoles` property, add the [roles](https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.Shared.Web/roles.json). Make sure that the role values are exactly matched to those defined, and assign a unique GUID value of your choice to each of the roles.
+If you want to update the manifest, see the details below:
 
     ```json
```

Comment:
>     [](start = 0, length = 4)

Can you move it to beginning of file? and add ``` in the end of the block.
Right now it's treated as part of block


```json
   {something}
```
rather than 
   ```json
     {something}

Reply:
![image](https://user-images.githubusercontent.com/1523833/100152389-7a128a80-2e57-11eb-8836-56fe81d0bc6a.png)

it looks like this to me.
and this is how I think it should be formatted
![image](https://user-images.githubusercontent.com/1523833/100152458-94e4ff00-2e57-11eb-8ac4-19455d2bdbbc.png)

Reply:
![image](https://user-images.githubusercontent.com/1523833/100152532-b219cd80-2e57-11eb-886d-dad5c12e372b.png) <-current
![image](https://user-images.githubusercontent.com/1523833/100152578-c362da00-2e57-11eb-987e-f2f1c06f36a7.png) <- how it should be

Reply:
updated!